pyext/src/rmsd_calculation.py

```diff
@@ -75,7 +95,7 @@ def get_pdbs_coordinates(path, idfile_A, idfile_B):
     return np.array(conform), masses, radii, models_name
 
 
-def get_rmfs_coordinates(path, idfile_A, idfile_B,
+def get_rmfs_coordinates(path, idfile_A, idfile_B,resolution=1,
```

Comment:
*Probably* OK here, but generally when adding new default arguments to a function, add them to the end so that you don't break existing callers of these functions.